katello_certs_tools/sslToolLib.py

```diff
@@ -95,11 +99,12 @@ def gendir(directory):
     "makedirs, but only if it doesn't exist first"
     if not os.path.exists(directory):
         try:
-            os.makedirs(directory, 0700)
-        except OSError, e:
-            print "Error: %s" % (e, )
+            os.makedirs(directory, '0700')
```

Comment:
`os.makedirs` does not accept string parameters (at least not in Python 3.6 and 3.7)